nuget/engine/nunit.engine.nuspec

```
@@ -34,37 +34,52 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
+    
     <file src="bin/net20/nunit.engine.dll" target="lib/net20" />
     <file src="bin/net20/nunit.engine.core.dll" target="lib/net20" />
     <file src="bin/net20/nunit.engine.api.dll" target="lib/net20" />
     <file src="bin/net20/testcentric.engine.metadata.dll" target="lib/net20" />
+    
     <file src="bin/agents/net20/nunit-agent.exe" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent.exe.config" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent-x86.exe" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit-agent-x86.exe.config" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit.engine.core.dll" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/nunit.engine.api.dll" target="contentFiles/any/agents/net20" />
     <file src="bin/agents/net20/testcentric.engine.metadata.dll" target="contentFiles/any/agents/net20" />
+    
     <file src="bin/agents/net40/nunit-agent.exe" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent.exe.config" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent-x86.exe" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit-agent-x86.exe.config" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit.engine.core.dll" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/nunit.engine.api.dll" target="contentFiles/any/agents/net40" />
     <file src="bin/agents/net40/testcentric.engine.metadata.dll" target="contentFiles/any/agents/net40" />
+    
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.dll" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.dll.config" target="contentFiles/any/agents/netcoreapp3.1" />
+    <file src="bin/agents/netcoreapp3.1/nunit-agent.runtimeconfig.json" target="tools/agents/netcoreapp3.1" />
```

```suggestion
    <file src="bin/agents/netcoreapp3.1/nunit-agent.runtimeconfig.json" target="contentFiles/agents/netcoreapp3.1" />
```

I think this one's been missed, Charlie. (I also don't know why we use contentFiles instead of tools here, but keeping it consistent seems safest! 😅)

---

I initially tried to keep it consistent, but I got failures when I did that. However, I can now think of another reason for that, so I'll try again. If this were for 4.0, I'd say just drop the whole `contentFiles` thing unless somebody can say why.